study.md

```diff
@@ -40,14 +40,17 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+mutate the original variable being called upon rather than just making a copy of it
+to work on.
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+class <classname>
```

Comment:
This declares a class but doesn't create an instance of it.  `class_instance = ClassName.new`.